samples/maps/demo/blend-colors/demo.js

```diff
@@ -4,7 +4,11 @@
         'https://code.highcharts.com/mapdata/custom/world.topo.json'
```

Comment:
I'm wondering - what would it look like with a Europe map instead of a world map with zoom?

Reply:
It would look like this: https://jsfiddle.net/BlackLabel/jaugx91p/

In the data, there are also values for Porto Santo Airport and some airports in Russia, which are not included in `europe.topo.json` map.

Reply:
Agreed, the world map works well for this data set